packages/rest/src/index.ts

```diff
@@ -30,7 +30,7 @@ export * from './rest.application';
 export * from './rest.component';
 export * from './rest.server';
 export * from './sequence';
-export * from './coercion/rest-http-error';
+export * from './rest.httpError';
```

Comment:
I find the suffix `httpError` weird. Either use `http-error`, or don't change the file name at all (moving it out of `coercion` is a good idea though).